backend/config/packages/security.yaml

```
@@ -48,13 +54,17 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        - { path: ^/api/login, roles: PUBLIC_ACCESS }
-        - { path: ^/api/auth/login, roles: PUBLIC_ACCESS }
-        - { path: ^/api/auth/register, roles: PUBLIC_ACCESS }
-        - { path: ^/api/csrf-token, roles: PUBLIC_ACCESS }
+        # Public API endpoints
+        - { path: ^/api/v1/auth/(login|register), roles: PUBLIC_ACCESS }
+        - { path: ^/api/v1/csrf-token, roles: PUBLIC_ACCESS }
+        - { path: ^/api/health, roles: PUBLIC_ACCESS }
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)